server/roomlogs.ts

```diff
@@ -16,6 +17,20 @@ interface RoomlogOptions {
 	noLogTimes?: boolean;
 }
 
+
+interface RoomlogRow {
+	type: string;
+	room: string;
+	user: string | null;
```

Comment:
Your schema says `room` and `userid`. Should these maybe just be consistently `roomid` and `userid`?